crash_alert/main_crash_alert.py

```diff
@@ -0,0 +1,89 @@
+from dotenv import load_dotenv
+from paho.mqtt import client as mqtt #import the client
+from crash_alert import CrashAlert
+import argparse
+import logging
+import os
+import json
+from mhp import topics
+
+class CrashAlertDriver():
+    """
+    Class that runs the crash alert. Listens to messages from the crash detection.
+    """
+    def __init__(self, host, cooldown, slack_webhook, port=1883):
+        """
+        Constructor for the class.
+
+        :param host: a string representing the MQTT host
+        :param cooldown: an integer representing the cooldown time in seconds for the crash alert
+        :param slack_webhook: a string representing the webhook url for the slack api
+        :param port: an optional integer representing the MQTT port, default to 1883
+        """
+        self.host = host
+        self.port = port
+        self.topic = str(topics.WirelessModule.id(3).crash_alert)
+        self.client = None
+        self.crash_alert = CrashAlert(cooldown, slack_webhook)
+        self.true_count = 0
+
+    @staticmethod
+    def get_args(argv=None):
+        """
+        Get arguments passed into Python script.
+
+        :param argv: an optional list of strings to manually input, by default None
+        :returns: a dictionary containing all arguments passed from command line
+        """
+        parser = argparse.ArgumentParser()
+        parser.add_argument("--host", type=str, default="localhost", help="ip address")
+        parsed_args = parser.parse_args(argv)
+        return parsed_args
+
+    def on_connect(self, client, userdata, flags, connection_result):
+        """
+        Subscribes the client to the given topic.
+        """
+        self.client.subscribe(self.topic)
+
+    def on_message(self, client, userdata, message):
+        """
+        Calls alert function in CrashAlert class to send alerts via APIs.
+        """
+        decoded_message = str(message.payload.decode("utf-8"))
+        payload = json.loads(decoded_message)
+        logging.info(f"MQTT message received: {payload}")
+        if payload["value"]:
+        # check for five true results
+        #     self.true_count += 1
+        # if self.true_count == 5:
+            return_msg = self.crash_alert.alert()
+            logging.debug(return_msg) 
+            # self.true_count = 0
```

Comment:
```suggestion
            return_msg = self.crash_alert.alert()
            logging.debug(return_msg) 
```
Unless really needed, we can remove the commented out code